sigmf-spec.md

```diff
@@ -293,8 +294,30 @@ the `global` object:
 |`recorder`|false|string|The name of the software used to make this SigMF recording.|
 |`license`|false|string|A URL for the license document under which the recording is offered; when possible, use the canonical document provided by the license author, or, failing that, a well-known one.|
 |`hw`|false |string|A text description of the hardware used to make the recording.|
+|`geolocation`|false|GeoJSON or string|Location of the recording system.|
+|`hagl`|false|double|Antenna height above ground level (in meters).|
 |`extensions`|false|object|A list of extensions used by this recording.|
 
+##### The `geolocation` Field
+The `core:geolocation` field in the `global` object is used to store the
+location of the recording system. This can be either a string (ISO 26324 DOI)
+to a separate recording of geolocation data or a GeoJSON (RFC 7946) geometry
+object as shown below. Point geometry objects are the only supported type for
+this field. Per the GeoJSON specification, the point coordinates are defined
+as `laitude`, `longitude` (required, in degrees), and `altitude` (optional, in
+meters above the WGS84 ellipsoid).
```

Comment:
I think we should add detail that makes it clear that:
* those fields must appear in that order
* they are all WGS-84
* this is defined by [RFC 5870](https://datatracker.ietf.org/doc/html/rfc5870), which GeoJSON implements

Also, you've got a typo one line up for `latitude`

Reply:
Agree (and thanks. damn gedit's lack of spell checking)